doc/schemas/listclosedchannels.schema.json

```diff
@@ -98,11 +98,11 @@
                 "items": {
                   "type": "string",
                   "enum": [
-                    "static_remotekey/even",
-                    "anchor_outputs/even",
-                    "anchors_zero_fee_htlc_tx/even",
-                    "scid_alias/even",
-                    "zeroconf/even"
+                    "static_remotekey_even",
+                    "anchor_outputs_even",
+                    "anchors_zero_fee_htlc_tx_even",
+                    "scid_alias_even",
+                    "zeroconf_even"
```

Comment:
Yeah, you can't do this, these are the actual strings we output.  An "enum" in the schema just means a subset of possible values: if the / is bad for Rust, it needs to change it there...

Reply:
I restored them back to their original state.
I'll wait for the feedback if this should be converted into an enum in primitives (like in my initial PR that I closed in favor of this one) or stay as string.